### ISSUE TITLE
Travis documentation suggests "Display [API token] in build log": seems insecure

### ISSUE THREAD

**Issue by mcomella** [danger/danger-js#675]
I was going through [the getting started guide](https://danger.systems/js/guides/getting_started.html) and saw this section for Travis:

![image](https://user-images.githubusercontent.com/759372/46239147-4efc8100-c349-11e8-8ed8-4a0d4c60c628.png)

In particular:
> If you have an open source project, you should ensure "Display value in build log" enabled, so that PRs from forks work.

If I understand correctly, this will insert your `DANGER_GITHUB_API_TOKEN`'s contents into the build log. For open source projects, build logs are often public, e.g. https://travis-ci.org/mozilla-mobile/focus-android/builds/434818832 This means anyone can find and then use your API token: I would recommend removing these lines.

It looks like this would need to be removed from the CLI init and the Travis documentation: https://github.com/danger/danger-js/search?q=display+value+in+build+log&unscoped_q=display+value+in+build+log

**Comment by kraftbj** [comment]
@mcomella I agree with you that it feels very wrong to expose the token like this. The side-effect is PRs from forks won't be able to have Danger comment on the PR. I'm not sure a better way to handle it off the top of my head, though.

**Comment by orta** [comment]
Yeah, I'm afraid those are the reasonable constraints from the CI host's perspective. 

An alternative is that you can use the [GitHub App for danger-js](https://github.com/danger/danger-js/blob/master/CHANGELOG.md#370) - which uses the GitHub App creds embedded inside danger, but it's pretty undocumented because I'm still not entirely sold on checks support, and have never used this myself on a project, but it should work
